src/views/SignUpView.vue

```diff
@@ -125,9 +105,42 @@ export default {
       isToastShown: false
     }
   },
+  computed: {
+    inputs() {
+      return [
+        {
+          model: this.userData.name,
+          type: 'text',
+          placeholder: 'User name',
+          conditions: this.isFormSubmitted && this.errorMessages.nameError !== '',
+          error: this.errorMessages.nameError
+        },
+        {
+          model: this.userData.name,
+          type: 'password',
+          placeholder: 'Password',
+          conditions: this.isFormSubmitted && this.errorMessages.nameError !== '',
+          error: this.errorMessages.passOneError
+        },
+        {
+          model: this.userData.name,
+          type: 'password',
+          placeholder: 'Repeat password',
+          conditions: this.isFormSubmitted && this.errorMessages.nameError !== '',
+          error: this.errorMessages.passTwoError
+        },
+        {
+          model: this.userData.name,
+          type: 'email',
```

Comment:
Is `this.userData.name` expected as model in all of these?

Reply:
Same for conditions ☝️

Reply:
**Corrected**